src/components/Categories.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Categories() {
+  return <div>This is check</div>;
+}
```

Comment:
- [ ] as part of the requirements kindly add a `<Button>` with the text "Check status" in your Categories page 🙏